libs/standard-tests/langchain_standard_tests/integration_tests/vectorstores.py

```diff
@@ -250,25 +250,8 @@ async def test_add_documents_with_ids_is_idempotent(
         await vectorstore.aadd_documents(documents, ids=["1", "2"])
         documents = await vectorstore.asimilarity_search("bar", k=2)
         assert documents == [
-            Document(page_content="bar", metadata={"id": 2}),
-            Document(page_content="foo", metadata={"id": 1}),
-        ]
-
-    async def test_add_documents_without_ids_gets_duplicated(
```

Comment:
Removed this unit test to allow vectorstores to implement assignment of IDs by document hash